examples/eks-quickstart-app-dev.yaml

```diff
@@ -0,0 +1,27 @@
+# An example of ClusterConfig object compatible with the "app-dev" quickstart
+# profile. See also: https://github.com/weaveworks/eks-quickstart-app-dev
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: cluster-1
+  region: eu-north-1
+
+nodeGroups:
+  - name: ng-1
+    instanceType: m5.large
+    minSize: 1
+    maxSize: 2
+    iam:
+      withAddonPolicies:
+        albIngress: true
+        autoScaler: true
+        cloudWatch: true
+
+cloudWatch:
```

Comment:
Let's not include these by default.

Reply:
FWIW, I did because they were in `examples/01-simple-cluster.yaml`. But sure, I'll remove these.